File: Day25/Solution.cpp
int main() {
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */
        int t;
        cin >> t;
        for (int i=0; i < t; i++){
            int flag = 0;
            int num;
            cin >> num;
            if (num < 2){
                cout << "Not prime\n";
            }
            else
            {
                for (int j=2; j <= sqrt(num); j++){
                    if (num%j == 0){
                        flag =1;
                        break;
                    }
                }
                if (flag == 0){
                    cout << "Prime\n";
                }
                else {
                    cout << "Not prime\n";
                }
            }

        }

    return 0;
}

File: Day25/Solution.go
package main
import(
    "fmt"
    "math"
)

func main() {
    //Enter your code here. Read input from STDIN. Print output to STDOUT
    var n,number int

    // take the inputs
    fmt.Scanf("%d", &n)
    for i:=0; i<n; i++ {
        fmt.Scanf("%d",&number)
        fmt.Println(checkPrime(number))
    }
}

func checkPrime(number int) string {
    if number == 1 {
        return "Not prime"
    }

    sq := int(math.Sqrt(float64(number)))
    for i:=2; i<sq+1; i++ {
        if number % i == 0 {
            return "Not prime"
        }
    }
    return "Prime"
}
